dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/WebContainer.java

```diff
@@ -191,8 +191,6 @@ public Object run() {
 
     }
 
-    protected static boolean decodePlusSign = true; //default to not decoding the plus sign per URL Spec
```

Comment:
Keep this one and use it via the getter so we can have a single point of access this variable.

Reply:
Updated per your comment.